### DISCUSSION TITLE
目次の前の別のページ

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1827] [Q&A]
目次の前にページを挿入するにはどうすればよいですか？

**Comment by kmuto** [comment]
catalog.ymlのPREDEF下で宣言したreファイルは、前付(frontmatter)として、目次よりも前に入ります。
(なお、前付には章番号は付きません。)

**Comment by jasondee1992** [comment]
ここにPREDEFヘッダーにテキストを入力する方法。

![image](https://user-images.githubusercontent.com/49851974/177707709-0a790a09-969b-4b73-985d-2e5992a3b724.png)

![image](https://user-images.githubusercontent.com/49851974/177707627-3024e4f1-3c79-4cdf-a899-1e63a58576ba.png)


![image](https://user-images.githubusercontent.com/49851974/177707947-fcdd048a-c5b6-481f-8e60-33ce4903bcf8.png)

**Comment by kmuto** [comment]
該当の箇所はheadings, 「柱」と呼ばれますが、確かに、jsbookのスタイルではfrontmatterに柱の内容が入らないですね。
「これは仕様であり、必要ならTeXの知識を得てスタイルファイルを調整する」というのが正式な道となりそうです。

雑でad-hocな方法としては、

```
= test

//embed[latex]{
\markboth{LEFT}{RIGHT}
//}

This is a paragraph, paragraph,
...
```

のようにすれば柱に文字は入ります(雑でad-hocな方法なので、おすすめしませんが)。

**Reply by jasondee1992** [reply]
LEFT」を単語に変更しようとしてエラーが発生しましたが、機能せず、「RIGHT」を削除します

`= test

//embed[latex]{
\markboth{テスト}
//}

This is a paragraph, paragraph,
...`

![image](https://user-images.githubusercontent.com/49851974/177713454-8241aaa6-b4b1-4c59-a554-ef9a9e5e928c.png)

**Reply by kmuto** [reply]
`\markboth` はその名前のとおり左と右の2つの引数が必要です。
いずれにせよRe:VIEWのembed命令は、そのバックエンド(ここではlatex)について知識がありさまざまなトラブルを自力解決できるということが前提になっているので、「柱の文字列を指定しないと仕事にならない」の重要度でないのなら、「柱が空でも気にしない」ことをお勧めします。
